File: CP_Assignment_1/ABD_8.c
// Assignment-1
// Roll No. - T014
// Q)8

#include <stdio.h>
int main()
{
    int arr[10],o=0,e=0,n=0;
    printf("Enter the (ten)elements of Array:\n");
    for(int i=0;i<10;i++)
    {
        printf("Array[%d]= ",i);
        scanf("%d",&arr[i]);
        (arr[i]%2==0) ? e++ : o++ ;
        if(arr[i]<0)
        	n++;
    }
    printf("There are\n%d Odd Elements.\n%d Even Elements.\n%d Negative Elements\n",o,e,n);
    
}

File: CP_Assignment_5/ABD_8.py
# Assignment-5
# Q)8

n = int(input("Enter the number : "))
print(n, end=" ")
while n != 1:
    if n % 2:
        n = (3 * n) + 1
    else:
        n = n // 2
    print(n, end=" ")
print()
